subset/bacnet/bacnetTests/src/main/java/PicsTest.java

```diff
@@ -0,0 +1,102 @@
+import com.google.common.collect.Multimap;
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.RemoteDevice;
+import com.serotonin.bacnet4j.npdu.ip.IpNetwork;
+import com.serotonin.bacnet4j.service.unconfirmed.WhoIsRequest;
+import helper.*;
+
+import java.util.Map;
+
+public class PicsTest {
+
+  private Connection connection;
+  private BacnetValidation validator;
+  private BacnetPoints bacnetPoints = new BacnetPoints();
+  private String testName = "protocol.bacnet.pic";
+  private String passedTestReport = String.format("RESULT pass %s\n", testName);
+  private String failedTestReport = String.format("RESULT fail %s\n", testName);
+  private String skippedTestReport = String.format("RESULT skip %s", testName);
```

Comment:
I'm 80% sure we need a `\n` at the end of the skip line too

Reply:
I've removed the "\n" for the description to be on the same line.

Reply:
I think the issue is, if there are _no_ appendices found in the `appendicesMap` then the newline never gets added and it breaks the output markdown:

https://github.com/JoelRedstone/daq/blob/bacnet_pics/subset/bacnet/bacnetTests/src/main/java/helper/Csv.java#L90

You could add the newline here too? :

https://github.com/JoelRedstone/daq/blob/bacnet_pics/subset/bacnet/bacnetTests/src/main/java/helper/Csv.java#L93